source/_integrations/smtp.markdown

```diff
@@ -83,115 +83,113 @@ A sample configuration entry for Google Mail.
 ```yaml
 # Example configuration.yaml entry
 notify:
-  - name: NOTIFIER_NAME
+  - name: "NOTIFIER_NAME"
     platform: smtp
-    server: smtp.gmail.com
+    server: "smtp.gmail.com"
     port: 587
     timeout: 15
-    sender: john@gmail.com
+    sender: "john@gmail.com"
     encryption: starttls
-    username: john@gmail.com
-    password: thePassword
+    username: "john@gmail.com"
+    password: "thePassword"
     recipient:
-      - james@gmail.com
-      - bob@gmail.com
-    sender_name: My Home Assistant
+      - "james@gmail.com"
```

Comment:
For another time, we should replace all email addresses like this with `john.smith@example.com` or similar